src/tools/fat_reader.c

```diff
@@ -6,178 +6,186 @@ typedef uint8_t bool;
 #define true 1
 #define false 0
 
-typedef struct{
-
-    //BIOS PARAMETER BLOCK for FAT12
-    uint8_t BootJumpInstruction[3];
-    uint8_t OemId[8];
-    uint16_t BytesPerSector;
-    uint8_t SectorsPerCluster;
-    uint16_t ReservedSectors;
-    uint8_t FATsNumber;
-    uint16_t RootDirEntries;
-    uint16_t TotalSectors;
-    uint8_t MediaType;
-    uint16_t SectorsPerFAT;
-    uint16_t SectorsPerTrack;
-    uint16_t HeadsCount;
-    uint32_t HiddenSectors;
-    uint32_t LargeSectors;
-
-    //EXTENDED BOOT RECORD
-    uint8_t DriveNumber;
-    uint8_t WindowsReserved;
-    uint8_t Signature;
-    uint8_t VolumeID[4];
-    uint8_t VolumeLabel[11];
-    uint8_t SystemID[8];
-
-}__attribute__((packed)) BootSector;
-
-typedef struct{
-    uint8_t FileName[8];
-    uint8_t FileExtension[3];
-    uint8_t Attributes;
-    uint16_t Reserved;
-    uint16_t CreationTime;
-    uint16_t CreationDate;
-    uint16_t LastAccessDate;
-    uint16_t Ignore;
-    uint16_t LastWriteTime;
-    uint16_t LastWriteDate;
-    uint16_t FirstLogicalCluster;
-    uint32_t FileSize;
-
-}__attribute__((packed)) DirEntry;
+typedef struct
+{
+
+    // BIOS PARAMETER BLOCK for FAT12
+    uint8_t bootJumpInstruction[3];
+    uint8_t oemId[8];
+    uint16_t bytesPerSector;
+    uint8_t sectorsPerCluster;
+    uint16_t reservedSectors;
+    uint8_t FAT_number;
+    uint16_t rootDirEntries;
+    uint16_t totalSectors;
+    uint8_t mediaType;
+    uint16_t sectorsPerFAT;
+    uint16_t sectorsPerTrack;
+    uint16_t headsCount;
+    uint32_t hiddenSectors;
+    uint32_t largeSectors;
+
+    // EXTENDED BOOT RECORD
+    uint8_t driveNumber;
+    uint8_t windowsReserved;
+    uint8_t signature;
+    uint8_t volumeID[4];
+    uint8_t volumeLabel[11];
+    uint8_t systemID[8];
+
+} __attribute__((packed)) BootSector;
+
+typedef struct
+{
+    uint8_t fileName[8];
+    uint8_t fileExtension[3];
+    uint8_t attributes;
+    uint16_t reserved;
+    uint16_t creationTime;
+    uint16_t creationDate;
+    uint16_t lastAccessDate;
+    uint16_t ignore;
+    uint16_t lastWriteTime;
+    uint16_t lastWriteDate;
+    uint16_t firstLogicalCluster;
+    uint32_t fileSize;
+
+} __attribute__((packed)) DirEntry;
 
 BootSector disk_BootSector;
-uint8_t* FATTable = NULL;
-DirEntry* disk_RootEntry = NULL;
+uint8_t *FAT_Table = NULL;
+DirEntry *disk_RootEntry = NULL;
```

Comment:
Star after variable type declaration
(convention)

Reply:
can we stay to type *name convenion because my autoformatter in vsc automatically does it?